CONTRIBUTING.md

```diff
@@ -243,6 +246,39 @@ cmake -S . -B ./build -D APP_my_application:BOOL=1
 cmake --build ./build -j
 ```
 
+
+### Adding a Workflow
+
+> NOTE: Workflows are _end-to-end_ reference applications that demonstrate complete pipelines from sensors to results. To determine if your contribution should be a workflow rather than an application, consider whether it represents a complete "from sensor to insight" pipeline. Workflows typically integrate multiple components (operators, models, etc.) to solve a specific use case in a production-ready manner, while applications may focus on demonstrating specific functionality or techniques.
```

Comment:
> production-ready manner

We should be careful about "production-ready" claims. Has the workflow gone through QA?

Reply:
That’s a good point. Will remove it. Thanks